tests/pdctl/keyspace/keyspace_group_test.go

```diff
@@ -96,7 +96,7 @@ func TestSplitKeyspaceGroup(t *testing.T) {
 	re.NoError(failpoint.Enable("github.com/tikv/pd/pkg/keyspace/acceleratedAllocNodes", `return(true)`))
 	re.NoError(failpoint.Enable("github.com/tikv/pd/server/delayStartServerLoop", `return(true)`))
 	keyspaces := make([]string, 0)
-	for i := 0; i < 500; i++ {
+	for i := 0; i < 129; i++ { // 128 is the default max txn ops limit in etcd.
```

Comment:
If we use 129 here, we'll add 129 txn ops instead of 128, right?

Reply:
you add 0, 1, ..., 128 keypsace here, 129 in total which will exceed the default max txn ops limit in etcd.

Reply:
yes, test the case which exceed the default max txn ops limit in etcd

Reply:
oh, could add this statement in comment? Without this statement, the code looks like have bug.